basket/settings.py

```diff
@@ -178,12 +179,18 @@ def path(*args):
 SECURE_CONTENT_TYPE_NOSNIFF = config('SECURE_CONTENT_TYPE_NOSNIFF', default=True, cast=bool)
 SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default=False, cast=bool)
 SECURE_REDIRECT_EXEMPT = [
-    r'^healthz/$',
-    r'^readiness/$',
+    r'^healthz/(ping/)?$',
```

Comment:
Changing the readiness URL would mean temporarily removing it prior to deploying the new code and then re-adding it afterward, otherwise we would end up with a [failed deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#failed-deployment) due to all of the pods failing the readiness probe at the old URL. Is there any way to preserve the existing readiness URL to eliminate the need for that coordination?

Reply:
I only just added the readiness URL to basket and it's not yet in prod. I'm sure we could make that happen for other projects that are already using readiness. The odd thing will be that for watchman the base URL is the full check (db, cache, etc.) and the `base/ping/` is the liveness, which I think if we use `/healthz/` as the base URL means that it would switch meanings. A small thing probably but potentially confusing for a bit. We can of course make it so that our old URLs work by simply not using their `urls.py` and adding their views directly to ours. I can't see why that wouldn't work just as well.

Reply:
Now that you've [changed the URLs](https://github.com/mozmeao/basket/pull/64/commits/4ac7bd14a1cf3e7e80e8743e4f94d224fce5153f) back you need to revert this.

Reply:
Ah! Great catch. Done.